### PR TITLE
added update endpoint

### PR DESCRIPTION
**Pull request by Pierregvx**
added the endpoints for the plugin update, 
the update function in implemented here #627

### CODE REVIEW

**Review by Thykof — CHANGES_REQUESTED** [review]

**Review by peterjah — CHANGES_REQUESTED** [review]
There is no need for a new endpoint.
just use `POST /plugin-manager/%s/execute` with the option "update"

**Review by peterjah — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Review by Thykof — CHANGES_REQUESTED** [review]
You need to change "update" to something else because now it's implemented https://github.com/massalabs/thyra/blob/main/api/test/robot_tests/plugin_manager/plugin_manager.robot#L181 ^^

File: api/test/robot_tests/plugin_manager/plugin_manager.robot [hunk 1]
```diff
@@ -178,7 +178,7 @@ POST /plugin-manager/register with invalid body
 
 POST /plugin-manager/{id}/execute with NotImplemented update command
```
Comment (peterjah) [inline api/test/robot_tests/plugin_manager/plugin_manager.robot:179]: with NotImplemented update command => with unknown command

**Comment by Thykof** [comment]
> You need to change "update" to something else because now it's implemented https://github.com/massalabs/thyra/blob/main/api/test/robot_tests/plugin_manager/plugin_manager.robot#L181 ^^

my bad, you can remove this test, test for invalid command already exists

**Review by Thykof — APPROVED** [review]

**Pull request merged**